src/components/PetsList.jsx

```diff
@@ -1,31 +1,36 @@
 import FilterList from "./FilterList";
 import PetCard from "./PetCard";
-import { PetItems } from "../files/PetsData";
 import React from "react";
+import usePets from "../hooks/usePets";
 
 const PetsList = () => {
+  const [{ petItems = [], isLoading }] = usePets("/hub/pets");
+
   return (
     <React.Fragment>
-      <div className="row">
-        <div className="col-md-9 col-xs-12">
-          {PetItems ? (
-            <div className="col">
-              <FilterList
-                items={PetItems}
-                renderItem={props => (
-                  <div key={props.id}>
-                    <PetCard {...props} />
-                  </div>
-                )}
-              />
+      <div className="container">
+        {isLoading ? (
+          <p>Loading Baltimore County News...</p>
```

Comment:
`News...` I supect this should be "Adoptable Pets" it might be worth asking matt about this.

Reply:
done